content/blog/gcp-v8-release/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Pulumi Google Cloud Provider Version 8.0.0"
+date: 2024-09-16T10:08:42-07:00
+draft: false
+meta_desc: "Release of the next version of the Pulumi Provider for Google Cloud"
+meta_image: meta.png
+authors:
+    - guineveresaenger
+tags:
+    - gcp
+    - release
+---
+
+The latest major release of the Pulumi Google Cloud Provider is available now!
+Our 8.0 release contains the latest upstream changes to keep you up-to-date along with the latest features and improvements from Pulumi.
+
+The Pulumi Google Cloud provider can be used to provision any of the Google Cloud resources available in the upstream provider.
+The provider is open source and available on GitHub so you can always follow along with current issues and developments, or even open your first pull request.
+
+<!--more-->
```

Comment:
Are you adding more?

Reply:
I thought this was the fold for the blog post format. Isn't the "more" anything below this intro?